examples/solids/qfunctions/finite-strain-neo-hookean-initial-2.h

```diff
@@ -355,7 +355,7 @@ CEED_QFUNCTION(ElasFSInitialNH2dF)(void *ctx, CeedInt Q,
                                       {q_data[7][i],
                                        q_data[8][i],
                                        q_data[9][i]}
-                                      };                              
+                                      };
```

Comment:
We should modify the top level Makefile so that `make style` reaches deep enough into these example files to catch this sort of thing for us.